src/content/locations/CityCouncil.coffee

```diff
@@ -32,7 +32,7 @@ add class Zoning extends Page
     -- My poor brain. I've never seen a room filled with such boredom, seeping out of the walls and oozing into me like some boring-slime.
 
   || bg="Council/Vibrator.jpg"
-    --> It's a good thing I brought a bunch of slaves to keep me entertained, trying not to squirm in the audience as their vibrators wiggle and thrust in and out. Combining a touch of magic with the Oooessay's sex toys is a promising field of study.
+    --> It's a good thing I brought a bunch of slaves to keep me entertained, trying not to squirm in the audience as their vibrators wiggle and thrust in and out. Combining a touch of magic with the Yousa's sex toys is a promising field of study.
```

Comment:
I actually intended to spell it differently every single time it comes up. Thanks for catching two places where I did it the same - I'll have to change one of them. :P